x86_build.bat

```diff
@@ -0,0 +1,3 @@
+Add MD5 to the build etc...
```

Comment:
[Weak/Broken Hash Algorithm](https://github.com/Microsoft/DevSkim/blob/master/guidance/DS126858.md)
A weak or broken hash algorithm was detected.

Suggestions:
**0** *+Add SHA256 to the build etc...*
**1** *+Add SHA512 to the build etc...*